test/unit/middleware/get_gateway_account_test.js

```diff
@@ -71,11 +71,28 @@ describe('middleware: getGatewayAccount', () => {
       done()
     })
   })
-  it('should extend the account data with supports3ds set to false if the account type is not worldpay', done => {
+  it('should extend the account data with supports3ds set to true if the account type is epdq', done => {
+    if (process.env.EPDQ_3DS_ENABLED === 'true') {
+      lodash.set(req, 'user.serviceRoles[0]', {gatewayAccountIds: ['1', '2', '3']})
+      const getGatewayAccount = setupGetGatewayAccount('1', 'epdq')
+      getGatewayAccount(req, res, next).then(() => {
+        expect(req.account).to.deep.equal({id: '1', payment_provider: 'epdq', supports3ds: true})
+        done()
+      })
+    } else {
+      lodash.set(req, 'user.serviceRoles[0]', {gatewayAccountIds: ['1', '2', '3']})
+      const getGatewayAccount = setupGetGatewayAccount('1', 'epdq')
+      getGatewayAccount(req, res, next).then(() => {
+        expect(req.account).to.deep.equal({id: '1', payment_provider: 'epdq', supports3ds: false})
+        done()
+      })
+    }
+  })
```

Comment:
Is it possible to create two tests for this? One where we enable the environment variable and one where we do not. Rather than having a single test that depends on the state of EPDQ_3DS_ENABLED to decide what get run.